.travis.yml

```diff
@@ -66,38 +69,35 @@ matrix:
     - env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.murmur=true'
     - env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true -Dtest.skip.murmur-ssl=true'
 
-addons:
-  coverity_scan:
-    # Coverity config parameters described in detail:
-    # https://scan.coverity.com/travis_ci
-    project:
-      name: "JanusGraph/janusgraph"
-      version: "0.2.0-SNAPSHOT"
-      description: "Scalable, distributed graph database"
-    notification_email: janusgraph-ci@googlegroups.com
-    build_command_prepend:
-      - if ! [ -v COVERITY_ONLY ]; then
-          echo "Skipping Coverity for non-Coverity job";
-          exit 0;
-        fi
-    build_command: mvn clean package -DskipTests=true --batch-mode
-    branch_pattern: coverity_scan
-
 install:
-  # Build and install current module and dependencies. Handle output timeouts and retry on error.
-  - if [ "${COVERITY_SCAN_BRANCH}" == 1 ] || [ -v COVERITY_ONLY ]; then
-      echo "Skipping install build step in Coverity branch/job";
+  - if [ "${TRAVIS_BRANCH}" == "${COVERITY_BRANCH_NAME}" ] && [ -v COVERITY_ONLY ]; then
```

Comment:
Will this run twice, once on each Linux and macOS? Looks like `-v` is not supported in the `bash` I have on macOS:

```bash
$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
```

The `-v` flag is not listed in the man page, and using it yields the error:

```
[: -v: unary operator expected
```

In any case, we don't need this to run on both platforms for Coverity; would just running it on Linux suffice?

Reply:
I'll switch to `-z ${COVERITY_ONLY:-}` instead. Build matrix doesn't vary OS (e.g. only dist Ubuntu Trusty is used).